projects/storefrontlib/src/lib/product/components/product-details/container/product-details-tabs.component.scss

```diff
@@ -0,0 +1,3 @@
+@import 'theme';
```

Comment:
yes, and actually, there's a whole bunch of old variables left over. let me clean that.

Reply:
I've cleaned up variables, but could drop the theme, as it was used in some function or mixin (which should't be the case)

Reply:
I was actually referring to the whole file since it's used in product-details component with the other scss for the component that has the same imports.